browser.js

```diff
@@ -47,7 +47,10 @@ not, since the functions are different and does
 not convert using browserify */
 function randomBytes(size) {
   var arr = new Uint8Array(size);
-  if (typeof window === 'undefined') {
+  if (
+    typeof browserCrypto === 'undefined' ||
```

Comment:
I don't think this is necessary (the fallback on line 7 is an empty object

Reply:
Good point. I have updated it 👍